elastic/kibana
# [Synthetics] adjust alerting content

## Summary

Resolves https://github.com/elastic/kibana/issues/155232

Updates the default alert copy.

A few things to note from the original ticket:

Recovery messages
---
1. Recovery due to deletion does not contain a link to the app, since we do not support viewing historic data after a monitor was deleted at this time
2. Recovery due to removing a location does not contain a link to the app, since we do not support viewing historic data from a removed location at this time
3. Recovery messages are a bit different than the ticket, to account for dynamic recovery reasons.

General
--
1. Step information is missing and will be addressed in another issue: https://github.com/elastic/kibana/issues/156773

### Testing

1. This is best tested with alert connectors. Ideally, set up a few alert connectors. At least try slack and server logs. You can set up your connectors at the Synthetics settings page
4. Create a down monitor. Wait for the rule to execute. View your connector. Inspect the content. Visit the view in app link
5. Wait for a bit
6. Update your monitor so it comes up. Wait for the rule to execute. View your connector. Inspect the content for the alert resolution. Visit the view in app link. Ensure the duration is correct
7. Update the monitor to set it down again. Wait for the alert to trigger
8. Remove the initial location and switch for a different location. Wait for the rule to trigger
9. Check your connector. Inspect the content
10. Set your monitor to a down state again.
11. Wait a bit
12. Delete your monitor. Inspect the content.
